File: doc/cli/examples/Core/Base/CxStopwatch_01.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace User
{
	partial class Examples
	{
		[XIE.CxPluginExecute]
		public void CxStopwatch_01()
		{
			XIE.CxStopwatch watch = new XIE.CxStopwatch();
			watch.Start();
			System.Threading.Thread.Sleep(100);
			watch.Stop();
			Console.WriteLine("Lap={0,9:0.000} msec, Elapsed={0,9:0.000} msec", watch.Lap, watch.Elapsed);
		}
	}
}

File: doc/cpp/examples/Core/Base/CxStopwatch_01.cpp

#include "xie_core.h"
#include <stdio.h>

namespace User
{

// ============================================================
void CxStopwatch_01()
{
	printf("%s\n", __FUNCTION__);

	xie::CxStopwatch watch;

	watch.Start();
	::Sleep(100);
	watch.Stop();
	printf("Lap=%9.3f msec, Elapsed=%9.3f msec\n", watch.Lap, watch.Elapsed);

	watch.Start();
	::Sleep(200);
	watch.Stop();
	printf("Lap=%9.3f msec, Elapsed=%9.3f msec\n", watch.Lap, watch.Elapsed);
}

}
